src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentPerDay = 40;
+  const advanceDiscount = 50;
+  const regularDiscount = 20;
+  const res = rentPerDay * days;
+
+  if (days > 2) {
+    return days > 6 ? res - advanceDiscount : res - regularDiscount;
```

Comment:
The ternary operator is concise, but according to the checklist, it's better to use `if` with `return` statements to simplify the conditions and avoid the use of `else`. Rewrite the conditions using `if` and `return` to improve readability.